### PR TITLE
Issue 4688: Handling znode deletion

### LINKED ISSUE #4688: znodes not getting deleted on Bookkeeper Cluster deletion
**Problem description**
The znodes are not deleted on Bookkeeper Cluster deletion since the `BK_LEDGERS_PATH` hasn't been updated.

**Problem location**
bookkeeper entrypoint.sh

### PR DESCRIPTION
**Pull request by SrishT**
Signed-off-by: SrishT <Srishti.Thakkar@dell.com>

**Change log description**  
The values of *BK_LEDGERS_PATH* and *BK_CLUSTER_ROOT_PATH* are modified to appropriate values so that znode deletion can be handled as expected.

**Purpose of the change**  
Fixes #4688 

**What the code does**  
Modifies the *BK_LEDGERS_PATH* and *BK_CLUSTER_ROOT_PATH* to `"/${PRAVEGA_PATH}/${BK_CLUSTER_NAME}/ledgers"` instead of the previous value which was dependent on the Pravega Cluster name as well i.e. `"/${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}/ledgers"`

**How to verify it**  
After the bookkeeper cluster is up and running, you can exec into a zookeeper pod and find the znodes created in the new path instead of the old one.

### CODE REVIEW

File: deployment/aws/installer/roles/install-bk/files/install-bk.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ PRAVEGA_CLUSTER_NAME=${PRAVEGA_CLUSTER_NAME:-"pravega-cluster"}
 BK_CLUSTER_NAME=${BK_CLUSTER_NAME:-"bookkeeper"}
 BK_AUTORECOVERY=${BK_AUTORECOVERY:-"false"}
 
-BK_LEDGERS_PATH="/${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}/ledgers"
+BK_LEDGERS_PATH="/${PRAVEGA_PATH}/${BK_CLUSTER_NAME}/ledgers"
```
Comment (pbelgundi) [inline deployment/aws/installer/roles/install-bk/files/install-bk.sh:21]: Why is this change needed? PRAVEGA_CLUSTER_NAME is passed as environment variable to Bookkeeper Cluster so bk-operator should be able to access and use this name.
This is a breaking change and is going to break all existing pravega installations on upgrade.

File: deployment/aws/installer/roles/install-bk/files/install-bk.sh [hunk 2]
```diff
@@ -48,8 +48,7 @@ until /opt/zk/zookeeper-3.5.1-alpha/bin/zkCli.sh -server $ZK_URL ls /; do sleep
 
 echo "create the zk root"
 /opt/zk/zookeeper-3.5.1-alpha/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}
-/opt/zk/zookeeper-3.5.1-alpha/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}
-/opt/zk/zookeeper-3.5.1-alpha/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}
+/opt/zk/zookeeper-3.5.1-alpha/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${BK_CLUSTER_NAME}
```
Comment (pbelgundi) [inline deployment/aws/installer/roles/install-bk/files/install-bk.sh:51]: We should not be using BK_CLUSTER_NAME. In cases where BK is not deployed on PKS it may not have a cluster name.

**Comment by medvedevigorek** [comment]
@SrishT / @pbelgundi is this change should be also ported to 0.7.1 release or not?

**Comment by SrishT** [comment]
Closing this PR as the issue been fixed in the bookkeeper operator itself: 
https://github.com/pravega/bookkeeper-operator/pull/28
